lib/user-interface/index.ts

```diff
@@ -33,71 +35,40 @@ export class UserInterface extends Construct {
 
     const appPath = path.join(__dirname, "react-app");
     const buildPath = path.join(appPath, "dist");
+    
+    const uploadLogsBucket = new s3.Bucket(this, "WebsiteLogsBucket", {
+        blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL,
+        removalPolicy: cdk.RemovalPolicy.DESTROY,
+        autoDeleteObjects: true,
+        enforceSSL: true,
+        
+      }
+      );
 
     const websiteBucket = new s3.Bucket(this, "WebsiteBucket", {
-      removalPolicy: cdk.RemovalPolicy.DESTROY,
-      blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL,
-      autoDeleteObjects: true,
-      websiteIndexDocument: "index.html",
-      websiteErrorDocument: "index.html",
+        removalPolicy: cdk.RemovalPolicy.DESTROY,
```

Comment:
review indentation

Reply:
fixed in commit: https://github.com/aws-samples/aws-genai-llm-chatbot/pull/310/commits/a67629d96c0d02eef93b223327fa4011c0316e14